portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +25,130 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    private List<String> countries;
+    private List<String> capitals;
+
+    @Override
+    public void init() {
+        countries = new ArrayList();
+        countries.add("Germany");
+        countries.add("Brazil");
+        countries.add("United States");
+        countries.add("France");
+        countries.add("Canada");
+        countries.add("Russia");
+        countries.add("China");
+        countries.add("Australia");
+        countries.add("Venezuela");
+        countries.add("India");
+        countries.add("Switzerland");
+        countries.add("Nigeria");
+        countries.add("Panama");
+        countries.add("Jamaica");
+        countries.add("South Africa (any one of the 3 capitals)");
+        countries.add("Italy");
+        countries.add("Mexico");
+        countries.add("Argentina");
+        countries.add("Colombia");
+        countries.add("Turkey");
+        
+        capitals = new ArrayList();
+        capitals.add("Berlin");
+        capitals.add("Brasilia");
+        capitals.add("Washington");
+        capitals.add("Paris");
+        capitals.add("Ottawa");
+        capitals.add("Moscow");
+        capitals.add("Beijing");
+        capitals.add("Canberra (yeah, I know...)");
+        capitals.add("Caracas");
+        capitals.add("New Delhi");
+        capitals.add("Bern");
+        capitals.add("Abuja");
+        capitals.add("Panama City");
+        capitals.add("Kingston");
+        capitals.add("Bloemfontein, Cape Town, Pretoria");
+        capitals.add("Rome");
+        capitals.add("Mexico City");
+        capitals.add("Buenos Aires");
+        capitals.add("Bogota");
+        capitals.add("Ankara");
+    }
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        // response.setContentType("text/html;");
+        // response.getWriter().println("<h1>Hello Murilo!</h1>");
+        //String[] pair = new String[2];
+        //response[0] = countries.get(4);
+        //response[1] = capitals.get(4);
+        Random rand = new Random();
+
+        response.setContentType("text/html;");
+        response.getWriter().println(countries.get(rand.nextInt(20)));
+    }
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
-  }
 }
+
+// @WebServlet("/country")
+// class Country extends HttpServlet {
```

Comment:
You can do this, but you need to put in in a .java file matching the class name.

Note on naming: This should probably be a `CountryServlet` (in `CountryServlet.java`) and not `Country` (which I would expect to model a single country instance, not a list/controller)